docs/ui/project-proposal.md

```diff
@@ -0,0 +1,36 @@
+# New Repository Proposal: ui
+
+## Summary
+
+This document proposes a new repository under the `instructlab` GitHub organization:
+
+- `instructlab/ui`
+
+## Background
+
+Currently we don’t have public repository under InstructLab that hosts UI-related work. The goal of this project is to provide a space for hacking a UI for the InstructLab. The scope of this project is to support the following personas through the UI:
+
+- External contributors who want to contribute skill and knowledge to the taxonomy repo
+- Triagers who will be evaluating the taxonomy contributions.
+
+These personas are not limited to the above list, but the initial focus will be on these two.
+
+Intent is to build this UI as a SaaS service for the upstream project and its taxonomy repo, but build in a way that someone could deploy the same service on their own infrastructure for managing their own taxonomy repo.
+
+## Maintainers
+
+The initial team of maintainers for this repository will be:
+
+- Anil Kumar Vishnoi <avishnoi@redhat.com>
+- Brent Salisbury <bsalisbu@redhat.com>
+- Taiga Nakamura <taiga@us.ibm.com>
+- Guang-Jie Ren <gren@us.ibm.com>
+- Juan Cappi <jmcappi@us.ibm.com>
+- Daniel Tan <chungtan@us.ibm.com>
+- Gregory Pereira <grpereir@redhat.com>
```

Comment:
Could we following the scheme of having a GitHub Team - say `UI Maintainers` - with these folks as the initial members?

Reply:
@nathan-weinberg I think we need the proposal to be first approved to go ahead and make the github team, otherwise it doesn't make sense to do that?. I personally would prefer to list the name of the initial maintainers, just to capture the diversity of the collaborators. But happy to fix it according to any policy we have.

Reply:
I think the suggestion is just to make your first sentence

> The initial team of maintainers (UI Maintainers) for this repository will be:

Reply:
Yeah the idea is that after the proposal is merged we could create said `UI Maintainers` GitHub Team and put all y'all on it - tighter permissions-wise and easier for adding/removing people later

Reply:
Got it. Fixed it. Thanks @russellb @nathan-weinberg for clarifying.